sklearn/impute/tests/test_impute.py

```diff
@@ -971,6 +971,59 @@ def test_iterative_imputer_catch_warning():
     assert not np.any(np.isnan(X_fill))
 
 
+@pytest.mark.parametrize("min_value, max_value",
+                         [(0, 100), (None, None), (-np.inf, np.inf),
+                          ((-5, 5, 10), (100, 200, 300)),
```

Comment:
```suggestion
                          (([-5, 5, 10], [100, 200, 300]),
```

Reply:
Committed changes to the test resolving this!